gtk/Gir.toml

```diff
@@ -742,6 +742,10 @@ status = "generate"
     name = "set_with_owner"
     #array with size
     ignore = true
+    [[object.function]]
+    name = "request_image"
+    # nullable callback parameter
+    ignore = true
```

Comment:
Can't this be configured via the toml file? If not, it can via the  gir file...

Please also send an MR upstream to gtk to fix the annotation :)

Reply:
Did you create an MR for gtk?

Reply:
No...

Reply:
I actually don't know what I'm supposed to update there.

Reply:
Add an annotation that the callback parameter is nullable. I usually don't merge any PRs like this until there's an upstream MR or at least an issue.